proto/api/v1/policy_category_service.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+option go_package = "v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/common.proto";
+import "api/v1/empty.proto";
+import "api/v1/search_service.proto";
+
+package v1;
+
+
+message PolicyCategory {
+    string id       = 1 ;
+    string name     = 2 ;
+    bool isDefault  = 3 ;
+}
+
+message PostPolicyCategoryRequest {
+    v1.PolicyCategory policyCategory = 1;
+}
+message GetPolicyCategoriesResponse {
+    repeated v1.PolicyCategory categories = 1;
+}
+
+// Note: Needed to rename these messages, since we cannot deprecate the existing category messages in policy_service.proto until 72
+// In 72, once older messages are deprecated, these messages will be renamed.
+message NewRenamePolicyCategoryRequest {
+    string id = 1;
+    string new_category_name = 2;
+}
+
+message NewDeletePolicyCategoryRequest {
+    string id = 1;
+}
+
+// PolicyCategoryService APIs can be used to manage policy categories.
+service PolicyCategoryService {
+
+    // GetPolicyCategory returns the requested policy category by ID.
+    rpc GetPolicyCategory (ResourceByID) returns (v1.PolicyCategory) {
+        option (google.api.http) = {
+            get: "/v1/policyCategories/{id}"
```

Comment:
Can we remove the capitalization? `policycategories` seems cleaner and more standard